Jonny-Riggs/Studio-Manager
# added info to readme

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git checkout JRbugs
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. 

## What you changed

## What was added to read me

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Have 2 teammates approved the PR?

Then Ship It!
